src/components/Restoration/BusinessName.vue

```diff
@@ -163,7 +163,8 @@ export default class BusinessName extends Mixins(CommonMixin, DateMixin, NameReq
     const nameRequest = await LegalServices.fetchValidContactNr(nrNum, phone, email)
     if (!nameRequest) throw new Error('Error fetching Name Request')
 
-    return nameRequest
+    // validateNameRequest() already throws printable errors
+    return this.validateNameRequest(nameRequest, this.requestActionCode)
```

Comment:
Doesn't this function call need phone and email parameters also?

Reply:
phone and email are optional arguments.